index.html

```diff
@@ -15,49 +15,92 @@
   <div class="container">
     <!--Start Header-->
     <header>
-      <img src="./assets/logo.svg" alt="logo meet">
+      <div class="header__logo">
+        <img  src="./assets/logo.svg" alt="logo meet">
+      </div>
       <section class="hero">
-        <div class="hero__img--left"></div>
+        <div class="hero__imgs-container ">
+          <div class="hero__imgs--left">
+            <div class="hero__imgs--up">
+              <img src="./images/avatar1.jpg" alt="avatar1" class="hero__img">
+              <img src="./images/avatar11.jpg" alt="avatar11" class="hero__img">
+              <img src="./images/avatar2.jpg" alt="avatar2" class="hero__img">
+            </div>
+            <div class="hero__imgs--up">
+              <img src="./images/avatar8.jpg" alt="avatar8" class="hero__img">
+              <img src="./images/avatar3.jpg" alt="avatar3" class="hero__img">
+              <img src="./images/avatar12.jpg" alt="avatar12" class="hero__img">
+            </div>
+          </div>
+        </div>
         <div class="hero__copy">
           <h1 class="hero__title">Group Chat for Everyone</h1>
           <p class="hero__text">
             Meet makes it easy to connect with others face-to-face virtually and collaborate across any device.
           </p>
           <div class="hero__btns">
-            <button class="hero__btn--left">Download v1.3</button>
+            <button class="hero__btn--left">Download <span class="hero__span"> v1.3</span></button>
             <button class="hero__btn--right">What is it?</button>
           </div>
+        </div>  
+        <div class="hero__imgs-container">
+          <div class="hero__imgs--right">
```

Comment:
Are you familiar with React.js? :)

Reply:
Yes, I am. 
I made about ten projects using React.js technology.  
My portfolio was made in React.js
https://my-react-portfolio-tatjana.vercel.app/work
And within it on the page My Work  are two projects that have also been done in React.js
1. Life in different countries...
2. Play it again, Sam!

Reply:
Well done! It looks great!

Reply:
Thank you